### ISSUE TITLE
LPI Canopy Cover Type Clarification

### ISSUE THREAD

**Issue by HarpoMark** [nationalparkservice/mojn-sv-database#20]
The database currently only allows an LPI Canopy Cover type to be declared for one stratum. For instance, you cannot say woody debris was present in the bottom AND middle stratum. I believe the index is enforcing this and want to make sure it's a correct rule.

**Comment by wright13** [comment]
I believe this is correct. I know that plant species only get recorded once per point except when one occurrence is dead and one is alive. I would double check with @hupp2it though.

**Comment by hupp2it** [comment]
This is correct. We discussed this a few times for IU, and we're deciding to follow the same strategy for SV. A species will only be recorded once for LPI, and it will be recorded in it's uppermost stratum. In your example, Mark, woody debris would be recorded in the middle stratum (also- yes, that is kind of a rare example, not sure how much woody debris we'd find more than 1 m above the ground).
